HerPublicWebsite.UnitTests/BusinessLogic/Services/QuestionFlowServiceTests.cs

```diff
@@ -414,13 +399,6 @@ public void RunForwardLinkTestCases(QuestionFlowServiceTestCase testCase)
                 country: Country.NorthernIreland
             ),
             QuestionFlowStep.IneligibleNorthernIreland),
-        new(
-            "Country continues to service unsuitable if the country is Other",
```

Comment:
Same as above - can you reach service unsuitable from the country page?
- If yes, should we have a different test case here?
- If no, is there a corresponding change to the `QuestionFlowService`?

Reply:
Removed all occurrences now 😊